app/src/lib/editors/win32.ts

```diff
@@ -32,13 +34,13 @@ interface IWindowsExternalEditor {
    * Some tools (like VSCode) may support a 64-bit or 32-bit version of the
    * tool - we should use whichever they have installed.
```

Comment:
![gracenote 1](https://user-images.githubusercontent.com/35483392/134619670-92cad53d-7694-42eb-ad61-5e555e76a682.jpg)

Reply:
https://github.com/desktop/desktop/pull/12778/files#diff-fc0d00865475b380cc1a9baab4efa7ea567dabe8b65ccb98aba1455b1cadba46https://github.com/notifications/beta?query=is%3Aunreadhttps://github.com/desktop/desktop/pull/12778/files#